### ISSUE TITLE
[FEATURE] Options for different color schemes/light & dark themes for accessibility reasons

### ISSUE THREAD

**Issue by frogsinasock** [CherpChat/IssueBoard#10]
The current color scheme of the cherpchat home page (White text on teal) makes it difficult for me to read the homepage/news and other features

An accessibility option to change the color scheme with a few different modes. For example classic (which could be the current theme) and light/dark themes

I suppose an alternative would be to just change the cherp color scheme all together 

Having custom color themes (if possible) may help with those who experience issues with eyestrain and such, like myself. But it is understandable how that could prove to be difficult and is not really a huge demand of mine.

**Comment by two-keys** [comment]
Yeah, sure. I can manage this. Custom themes'll still be locked behind the donation goal, but I can add in some non-custom light/dark themes after consulting with some folk about what light/dark color schemes would work for accessibility.
